src/d3a/d3a_core/sim_results/file_export_endpoints.py

```diff
@@ -64,10 +66,9 @@ def _populate_plots_stats_for_supply_demand_curve(self, area):
                 self.cumulative_offers[area.slug][market.time_slot] = {}
                 self.cumulative_bids[area.slug][market.time_slot] = {}
                 self.clearing[area.slug][market.time_slot] = {}
-            self.cumulative_offers[area.slug][market.time_slot] = market.state.cumulative_offers
```

Comment:
Sorry to only realise this now. The `MarketClearingState` is not used any more, right? Let's get rid of it then.

Reply:
Darn, ignore me